src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,15 +1,32 @@
 import React, { forwardRef } from 'react';
-import PropTypes from 'prop-types';
 import { bem } from '../../utils';
 import { Text } from '../Text';
 import styles from './Checkbox.scss';
 import { CHECKBOX_VIEWBOX } from '../../constants';
 
+interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
+    /** A unique id to reference this checkbox */
+    id: string;
+    /** If the checkbox should be disabled */
+    disabled?: boolean;
+    /** The label for the checkbox */
+    children?: string | React.ReactElement;
```

Comment:
I'm not sure how we should support this. Normally I think Checkbox is asking for a single node and this will have children as an array. The main thing is that we are adding a wrapper around the child if it is a string, otherwise not. In this case it is 2 children and the expected behaviour is not clear to me. See lines 21-28 in the same file. What would you expect to happen there?
I'm not sure how this passed validation so far, but I think we should stick with a single node as child. So you could wrap the whole thing in a `<span>` and then the current implementation works.